locale/tr_TR.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -0,0 +1,515 @@
+# PalMA translation.
+# Copyright (C) 2017 Universitätsbibliothek Mannheim
+# This file is distributed under the same license as the PalMA package.
+# Stefan Weil <stefan.weil@bib.uni-mannheim.de>, 2014.
+# Dennis Müller <dennis.mueller@bib.uni-mannheim.de>, 2017.
```

Comment:
Here you may replace my name and Dennis Müller by your own name and e-mail address.